File: 0344-reverse-string/0344-reverse-string.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        leng = len(s)
        mid = leng/2
        leng -= 1
        index = 0
        while(index < mid):
            x = leng - index
            temp = s[x]
            s[x] = s[index]
            s[index] = temp
            index += 1
        
        

File: 0344-reverse-string/0344-reverse-string.java
class Solution {
    public void reverseString(char[] s) {
        int len = s.length;
        int mid = len/2;
        len--;
        int index = 0;
        char temp;
        int x;
        while(index < mid){
            x = len - index;
            temp = s[x];
            s[x] = s[index];
            s[index] = temp;
            index++;
        }
    }
}